Resource.js

```diff
@@ -280,9 +280,16 @@ module.exports = function(app, route, modelName, model) {
       var findQuery = {};
 
       var getValue = function(value, param) {
+        var parsedValue = parseInt(value, 10);
+
         if (param.instance === 'Number') {
-          return parseInt(value, 10)
+          return parsedValue;
+        }
+
+        if (parsedValue) {
+          return new Date(parsedValue);
```

Comment:
Won't this end up converting ALL number parameters into a Date object?

Reply:
I added instance check, please take a look.